example/use_cases/employer_university_flow/employer_university_flow/util.go

```diff
@@ -0,0 +1,315 @@
+// For more information, please go to here:
+// https://identity.foundation/presentation-exchange/#presentation-request
+// and for the source code with the sdk,
+// https://github.com/TBD54566975/ssi-sdk/blob/main/credential/exchange/request.go
+// is appropriate to start off with.
+func makePresentationRequest(jwk cryptosuite.JSONWebKey2020, presentationData exchange.PresentationDefinition, targetId string) (pr []byte, signer *cryptosuite.JSONWebKeySigner, err error) {
+
+	cw.WriteNote("Presentation Request (JWT) is created")
+
+	// Signer:
+	// https://github.com/TBD54566975/ssi-sdk/blob/main/cryptosuite/jsonwebkey2020.go#L350
+	// Implements: https://github.com/TBD54566975/ssi-sdk/blob/main/cryptosuite/jwt.go#L12
+	signer, err = cryptosuite.NewJSONWebKeySigner(jwk.ID, jwk.PrivateKeyJWK, cryptosuite.Authentication)
+	if err != nil {
+		return
+	}
+
+	// Builds a presentation request
+	// Requires a signeer, the presentation data, and the target
+	// Target is the Audience Key
+	requestJWTBytes, err := exchange.BuildJWTPresentationRequest(*signer, presentationData, targetId)
+	if err != nil {
+		return
+	}
+
+	return requestJWTBytes, signer, err
+}
+
+func MakePresentationRequest(jwk cryptosuite.JSONWebKey2020, presentationData exchange.PresentationDefinition, targetId string) (pr []byte, signer *cryptosuite.JSONWebKeySigner, err error) {
+	return makePresentationRequest(jwk, presentationData, targetId)
+}
+
+// normalizePresentationClaims takes a set of Presentation Claims and turns them into map[string]interface{} as
+// go-JSON representations. The claim format and signature algorithm type are noted as well.
+// This method is greedy, meaning it returns the set of claims it was able to normalize.
+func normalizePresentationClaims(claims []exchange.PresentationClaim) []exchange.NormalizedClaim {
+	var normalizedClaims []exchange.NormalizedClaim
+	for _, claim := range claims {
+		ae := util.NewAppendError()
+		claimJSON, err := claim.GetClaimJSON()
+		if err != nil {
+			ae.Append(err)
+		}
+		claimFormat, err := claim.GetClaimFormat()
+		if err != nil {
+			ae.Append(err)
+		}
+		if ae.Error() != nil {
+			logrus.WithError(ae.Error()).Error("could not normalize claim")
+			continue
+		}
+		var id string
+		if claimID, ok := claimJSON["id"]; ok {
+			id = claimID.(string)
+		}
+		normalizedClaims = append(normalizedClaims, exchange.NormalizedClaim{
+			ID:             id,
+			Data:           claimJSON,
+			Format:         claimFormat,
+			AlgOrProofType: claim.SignatureAlgorithmOrProofType,
+		})
+	}
+	return normalizedClaims
+}
+
+func BuildPresentationSubmission(presentationRequest []byte, signer cryptosuite.Signer, verifier cryptosuite.JSONWebKeyVerifier, vc credential.VerifiableCredential) ([]byte, error) {
+	return buildPresentationSubmission(presentationRequest, signer, verifier, vc)
+}
+
+// https://github.com/TBD54566975/ssi-sdk/blob/d279ca2779361091a70b8aa3c685a388067409a9/credential/exchange/submission.go#L126
```

Comment:
maybe we should use relative references? meaning `/credential/exchange/submission.go`